docs/production/deployment.md

```diff
@@ -538,6 +538,19 @@ the system and deployment; `/etc/zulip/settings.py` is used to
 configure the application itself. The `zulip.conf` sections and
 settings are described below.
 
+When a setting refers to "set to true" or "set to false", the
+following values are ones considered to be "true", case-insensitively:
+
+- 1
+- y
+- t
+- yes
+- true
+- enable
+- enabled
+
+Any other value (including the empty string) is considered false.
```

Comment:
Do we want to specify that "true" is preferred?  I suppose we could standardize on `true` and replace the other values if they appear with a migration tool down the line.

Reply:
I've added some wording that `true` and `false` are canonical.